src/lib/utils/http-params.ts

```diff
@@ -40,3 +37,22 @@ export function createHttpParams(params:Params, arrayAsArray:boolean = false):Ht
     });
     return searchParams;
 }
+
+/**
+ *  Gets Params from an array
+ * @param params
+ * @param key
+ * @param searchParams
+ */
+function getParamsFromArray(params:Params, key:string, searchParams:HttpParams):HttpParams
+{
+    (params[key] as Array<any>).forEach((arrayItem:any) =>
+    {
+        if(arrayItem === null)
```

Comment:
Vielleicht auch noch undefined abfangen? Sonst haben wir ja das gleiche Problem.
```suggestion
        if(arrayItem === null || arrayItem === undefined)
```